notebooks/concepts/Data interactions through CubeViz.ipynb

```diff
@@ -0,0 +1,304 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from specutils.manipulation import box_smooth\n",
+    "\n",
+    "smoothed_spec = box_smooth(spec, 5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once we have the results from our manipulation/analysis function, we can insert the data into the appropriate viewer using the `add_data` method on the `app` object and passing in the requisite viewer reference."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This is necessary since the wcs is not being auto-linked in the manga data. If\n",
+    "#  we don't unset the previous data set, the new data won't be shown on the plot.\n",
+    "app.unset_viewer_data('spectrum-viewer', '6de4c8ee5659e87a302e3de595074ba5[FLUX]')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app.add_data(smoothed_spec, name=\"Smooth Spectrum\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app.set_viewer_data('spectrum-viewer', 'Smooth Spectrum')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Retriving manipulated data products\n",
+    "\n",
+    "Here, we'll go back up to the CubeViz application and use the collapse plugin to create a 2D data product from within the application. Afterward, we plot it in the `flux-viewer` viewer and then we can retrieve and view the data below in both the glue-jupyter image viewer as well as retrieving and the data and showing it in a matplotlib plot."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<b>User Task:</b> In the CubeViz application at the top, select the \"Collapse\" button in the toolbar and collapse the \"manga-7495-12704-LOGCUBE[FLUX]\" data set along the 0th axis to create a 2D image.\n",
+    "    \n",
+    "Next, in the top-left viewer (our `flux-viewer`), uncheck any currently plotted data and scroll to the bottom to click the checkbox next to \"Collapsed manga-7495-12704-LOGCUBE[FLUX]\" to plot the 2D image.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Show the current flux viewer\n",
+    "app.get_viewer('flux-viewer').show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "# Note that this returns a raw glue data object since we did not specific e.g. `cls=CCData`.\n",
```

Comment:
Hmm. I think we may *want* it to come out as `CCData` here for the use case shown.  Perhaps that should also be a default type like we've talked about (and maybe implemented here?) for `Spectrum1D`?

(this could go in a later PR if it's too much work to do now, but if so should be in a separate issue)

Reply:
Done.